.travis.yml

```diff
@@ -10,19 +10,23 @@ cache: pip
 dist: xenial
 
 env:
-  TOXENV=py,check,type
+  TOXENV=py,check,type,coverage
 
 jobs:
   include:
-  - name: "Tests: python3.6"
-    python: 3.6
-  - name: "Tests: python3.7"
-    python: 3.7
+    - name: "Tests: python3.6"
+      python: 3.6
+    - name: "Tests: python3.7"
+      python: 3.7
 
 install:
-- pip install -U pip setuptools
-- pip install -U tox==3.12.0
-- tox --notest
+  - pip install -U pip setuptools
+  - pip install -U tox==3.12.0
+  - pip install -U codecov
```

Comment:
Can we run codecov within tox https://github.com/codecov/example-python#testing-with-tox?

Reply:
Yes, but there's really no reason to. The only place we'd ever run `codecov` is Travis, so I'd prefer to keep it isolated to that environment (as opposed to putting the dep in `requirements-dev.txt` and adding possibly confusing env in `tox.ini`)

Reply:
Okay makes sense